src/main/typescript/constants/fileUploadConstants.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum FileUploadConstants {
+    FILE_UPLOAD_IN_PROGRESS = "FILE_UPLOAD_IN_PROGRESS",
```

Comment:
Isn't his confusing with https://github.com/DANS-KNAW/easy-deposit-ui/blob/cf27bd16162309c9079eb2e64b9ff1fe1716cb3a/src/main/typescript/components/form/parts/fileUpload/upload/FileLoader.tsx#L23

Reply:
`FILE_UPLOAD_IN_PROGRESS` is used only in the context of Redux, whereas `PROCESSING` is only used inside the `FileLoader.tsx` file. Note that the latter enum is also not exported, so it is not possible to access or leak it to the outside world. Hence, while I agree with you that the names are quite alike, I don't think there will be any confusion between them.